odl/discr/discretization.py

```diff
@@ -316,6 +316,26 @@ def __setitem__(self, indices, values):
         else:
             self.ntuple.__setitem__(indices, values)
 
+    @property
+    def restriction(self):
+        """The restriction operator associated with this vector.
+
+        See Also
+        --------
+        RawDiscretization.restriction : For full description
+        """
+        return self.space.restriction(self.ntuple)
```

Comment:
My fault here, I simply copied from `extension`. I suppose having one of these for `restriction` does not make much sense.